redaxo/src/addons/phpmailer/lib/cronjob.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * PHPMailer Addon.
+ *
+ * @author markus[dot]staab[at]redaxo[dot]de Markus Staab
+ *
+ * @package redaxo\phpmailer
+ */
+
+class rex_cronjob_mailer_purge extends rex_cronjob
+{
+    /**
+     * purgeMailarchive.
+     *
+     * @return int
+     */
+    protected function purgeMailarchive(int $days = 7, string $dir = '', int $log = 0)
```

Comment:
```suggestion
    private function purgeMailarchive(int $days = 7, string $dir = '', int $log = 0): int
```

Der Return-Type kann nativ notiert werden. Der andere Kommentar bringt keine interessante Zusatzinfo.
Methoden sollten per default private sein. protected nur, wenn man wirklich weiß, dass man es braucht.

Methode am besten nach ganz unten. Reihenfolge in Klassen sollte public, protected, private sein. Setzen wir bisher nicht per cs-fixer durch, weil es sehr viel Code verschieben würde im Altcode.